start.md

```diff
@@ -119,7 +119,7 @@ All provisioned systems need to be RAPIDS capable. Here's what is required:
 
 - <i class="fas fa-check-circle text-purple"></i> [CUDA 11.2](https://developer.nvidia.com/cuda-11.2.0-download-archive){: target="_blank"} with Driver 460.27.03 or newer
 - <i class="fas fa-check-circle text-purple"></i> [CUDA 11.4](https://developer.nvidia.com/cuda-11-4-0-download-archive){: target="_blank"} with Driver 470.42.01 or newer
-- <i class="fas fa-check-circle text-purple"></i> [CUDA 11.5](https://developer.nvidia.com/cuda-11-5-0-download-archive){: target="_blank"} with Driver 495.29.05 or newer
+- <i class="fas fa-check-circle text-purple"></i> [CUDA 11.8](https://developer.nvidia.com/cuda-11-8-0-download-archive){: target="_blank"} with Driver 520.61.05 or newer
```

Comment:
We are still supporting CUDA 11.5, so I think you leave 11.5 and add 11.8 here.

Reply:
Good call. I always forget this because we dropped 11.5 from our PR CI. I think it’s still tested in nightlies?

Reply:
Fixed in d7fb44346379c3542d2df53566c62e631a11ebf7.